microsoftgraph/microsoft-graph-explorer-v4
# Fix: Toolkit component scrolling

## Overview
Fixes #1864

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
When tabbing through the toolkit component, the toolkit scrolls to expose a new view. When this happens, the query response tabs are pushed upwards. 

## Testing Instructions

* How to test this PR
Check out this PR and run it locally
Load the app on a small screen(15 inches and below)
Click on the 'my profile' sample query/ Paste this query on the query textbox: https://graph.microsoft.com/v1.0/me
Click on the Toolkit component and tab through it until the focus gets to the js/css editor.
Notice that the toolkit does not make the query response tabs disappear
